traitcuration/static/styles/layout/header/_header.scss

```diff
@@ -0,0 +1,25 @@
+@import "../../utilities/variables";
+
+.header {
+  background-color: $color-primary !important;
+  padding: 0 15px;
```

Comment:
I think there are two competing (but also complementary) approaches to use either `px` (absolute) or `rem` (relative) values for sizes of elements. It looks like there are benefits and drawbacks for all of them (I found a good summary here: https://stackoverflow.com/questions/11799236/should-i-use-px-or-rem-value-units-in-my-css), so I'll leave it up to you which way to choose. My understanding that currently we're going with pixels

Reply:
In my experience, em for font-sizes and px for the rest works well and responsively. Of course if any problem about that arises I can tweak the CSS as needed

Reply:
Actually I have given it some more thought. I decided to use rem globally (except for box-shadow properties) to maintain consistency across the CSS code. I think however that we should first merge the open PRs and then I can change it so that I can avoid as many merge conflicts as possible. Do you agree?

Reply:
Yes, absolutely, this sounds sensible. I've created a separate issue to handle this: https://github.com/EBIvariation/trait-curation/issues/40

If you eventually decide that it's not worth it, then this is also fine. In this case just close that issue without changing anything.

Reply:
Yes, absolutely, this sounds sensible. I've created a separate meta-issue to track cases like this, when there are tiny things to change and it doesn't make sense to postpone merging otherwise good PRs: https://github.com/EBIvariation/trait-curation/issues/40